erpnext/accounts/doctype/purchase_invoice/purchase_invoice.js

```diff
@@ -382,21 +382,11 @@ cur_frm.fields_dict['items'].grid.get_field("item_code").get_query = function(do
 
 cur_frm.fields_dict['credit_to'].get_query = function(doc) {
 	// filter on Account
-	if (doc.supplier) {
```

Comment:
Same filter is there in sales_invoice.js as well. Change it there as well for consistency.